File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.py
# Last updated: 09/07/2026, 10:10:58
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        for j in range(1,len(nums)):
            if nums[i]!= nums[j]:
                i = i + 1
                nums[i] = nums[j]
        return i + 1    

        

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 09/07/2026, 10:10:52
class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        for(int j = 1;j<nums.length;j++){
            if(nums[i] != nums[j]){
                i+=1;
                nums[i] = nums[j];
            }
        }
        return i+1;
    }
}